gym_xiangqi/envs/__init__.py

```diff
@@ -1 +1 @@
-from gym_xiangqi.envs.xiangqi_env import XiangQiEnv
+from gym_xiangqi.envs.xiangqi_env import XiangQiEnv  # NOQA
```

Comment:
I am just curious. Does that comment tell flake8 to ignore that line?

Reply:
Yeap since on surface this looks like an unused import, flake8 will flag this if I don't add that comment.